### ISSUE TITLE
[Video Support][Android] Video keeps playing even when out of sight.

### ISSUE THREAD

**Issue by SaySayTakamura** [bluesky-social/social-app#5330]
**Describe the bug**

Once a video starts it never stops, even when you go into Full-Screen and pause it, once you quit the full-screen mode it will start playing from where it was paused.

*To Reproduce**

1. Let a video load and play by itself
2. Scroll up and down
3. It doesn't stop

OR
 
1. Let a video load and play by itself
2. Pause it from the full-screen mode
3. Quit from the full-screen 
4. It doesn't stop

**Expected behavior**

Whenever a video starts playing, it should pause itself once it's out of the viewer's sight or remain paused if it was paused from the full-screen mode.

Note: it would be nice to pause the video without the need to enter full-screen mode. (Tap and Hold to enter full-screen mode and just a tap to pause/play the video.

**Details**

 - Platform: Android
 - Platform version: Android 10
 - App version: 1.91.0 (Latest)

**Comment by scottdotweb** [comment]
I was just about to open an issue asking for video to stop playing while you compose a repost, but I think it's a case which falls under this.

**Comment by SaySayTakamura** [comment]
> I was just about to open an issue asking for video to stop playing while you compose a repost, but I think it's a case which falls under this. 

Yeah, i totally forgot of this kind of event, it should pause the video whenever the video post is not focused (cases such a modal appearing, tapping the post to go to the comment section (unpauses it once the post page load again) and many others).

**Comment by Skaldebane** [comment]
Same here, it keeps playing even after locking the screen.
